OpenRA.Mods.Common/Activities/Move/Move.cs

```diff
@@ -57,10 +58,12 @@ public Move(Actor self, CPos destination)
 				return path;
 			};
 			this.destination = destination;
+			this.targetLineColor = targetLineColor;
 			nearEnough = WDist.Zero;
 		}
 
-		public Move(Actor self, CPos destination, WDist nearEnough, Actor ignoreActor = null, bool evaluateNearestMovableCell = false)
+		public Move(Actor self, CPos destination, WDist nearEnough, Actor ignoreActor = null, bool evaluateNearestMovableCell = false,
+					Color? targetLineColor = null)
```

Comment:
That's two tabs more than we usually use in such cases.

Reply:
Fixed.